specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/settings.json

```diff
@@ -137,9 +137,6 @@
               }
             }
           },
-          "x-ms-pageable": {
```

Comment:
During a different sync meeting later this afternoon, I think it was @vickm who suggested that maybe this should be pageable initially, which means the response needs a `nextLink`. Effectively, how the generated client SDK methods look will be very different - and breaking if changed - if this is pageable or not. So if there's even a remote chance of paging in the future, it needs to be pageable now.

Reply:
Ok, we all just synced up and decided that it will never be pageable. The max number of items that it could ever even get to right now is 6 and we still don't even foresee that happening.